src/index.html

```diff
@@ -7,6 +7,37 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
```

Comment:
should work with `31` days in a month as well
![Screenshot 2024-01-15 at 19 14 38](https://github.com/mate-academy/layout_calendar/assets/117587463/cab58f29-fb30-4e79-9faa-15c18c4fcc92)